### PR TITLE
Moves methods to class-level and makes them static

### LINKED ISSUE #2: Move PHPDoc to class level
The PHPDoc at the following line should be moved to class level: https://github.com/Intervention/image-laravel/blob/eb6f2478cb646eadfbc32941ef92647f5c4893ac/src/Facades/Image.php#L12

Because they are now at method level (which is strange for `@method` PHPDoc), they are not used by the IDE.

### PR DESCRIPTION
**Pull request by diamondobama**
This PR resolves https://github.com/Intervention/image-laravel/issues/2 and adds more tests for 100% code coverage.

### CODE REVIEW

File: tests/FacadeTest.php [hunk 1]
```diff
@@ -4,23 +4,74 @@
...
+        $input = UploadedFile::fake()->image('image.jpg');
+        $result = Image::read($input);
+        $this->assertInstanceOf(ImageInterface::class, $result);
+    }
+
+    public function testThrowsExceptionWhenReadingNonExistentImage(): void
+    {
+        $this->expectException(Error::class);
+        $input = 'path/to/non_existent_image.jpg';
+        Image::read(null, $input);
```
Comment (olivervogel) [inline tests/FacadeTest.php]: Is it possible that this is a mistake? Either a non-existent file is being checked here, in which case the call should be made without `null`. Or do you want to check an incorrect call? Then the method should be renamed.

File: src/Facades/Image.php [hunk 2]
```diff
@@ -7,13 +7,13 @@
...
- * @method \Intervention\Image\ImageManager static create(int $width, int $height)
- * @method \Intervention\Image\ImageManager static animate(callable $callback)
+ * @method static \Intervention\Image\Interfaces\ImageInterface read(mixed $input, string|array|\Intervention\Image\Interfaces\DecoderInterface $decoders = [])
+ * @method static \Intervention\Image\Interfaces\ImageInterface create(int $width, int $height)
+ * @method static \Intervention\Image\Interfaces\ImageInterface animate(callable $callback)
  */
 class Image extends Facade
 {
-    protected static function getFacadeAccessor()
+    protected static function getFacadeAccessor(): string
```
Comment (olivervogel) [inline src/Facades/Image.php]: Doesn't that create an incompatibility with `Illuminate\Support\Facades\Facade::getFacadeAccessor()`?

**Review by olivervogel — COMMENTED** [review]
Thank you very much. This can be integrated, I have only two minor comments/questions.

File: tests/FacadeTest.php (hunk 1 above)
Comment (diamondobama) [reply tests/FacadeTest.php]: Thanks for catching that. I initially used `invoke()` from reflection, switched to the facade, and forgot to remove the `null` argument.

File: src/Facades/Image.php (hunk 2 above)
Comment (diamondobama) [reply src/Facades/Image.php]: This shouldn't be a problem, as the `Illuminate\Support\Facades\Facade::getFacadeAccessor()` return type is inferred from PHPDoc. However, I've removed it just to be safe.

**Comment by olivervogel** [comment]
Thanks again.

**Pull request merged**